src/video/emscripten/SDL_emscriptenevents.c

```diff
@@ -531,6 +531,11 @@ EM_BOOL
 Emscripten_HandleResize(int eventType, const EmscriptenUiEvent *uiEvent, void *userData)
 {
     SDL_WindowData *window_data = userData;
+
+    /* check if pixel ratio has changed */
+    double pixel_ratio = emscripten_get_device_pixel_ratio();
+    window_data->pixel_ratio = pixel_ratio;
+
```

Comment:
Sorry, wasn't very clear. ` window_data->pixel_ratio = emscripten_get_device_pixel_ratio()` should do. Also, the comment should probably be updated to something like "update pixel ratio".